.github/workflows/backport-pr.yaml

```diff
@@ -0,0 +1,26 @@
+name: Backport PR to release branch
+on:
+  pull_request_target:
+    types:
+      - closed
+      - labeled
+
+jobs:
+  backport:
+    name: Backport
+    runs-on: ubuntu-latest
+    # Only react to merged PRs for security reasons.
+    # See https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request_target.
+    if: >
+      github.event.pull_request.merged
+      && (
+        github.event.action == 'closed'
```

Comment:
I'm not entirely sure why this condition is here when `github.event.pull_request.merged` is checked, but it looks like this is coming directly from https://github.com/tibdex/backport/blob/main/.github/workflows/backport.yml

Reply:
TBH it is not entirely clear to me either. I'm taking a leap of faith that the action authors have created something that'll work as I expect :crossed_fingers: